start-proxy.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+set -e
+
+./gradlew ${DATABASE?}${MODE?}ComposeBuild
```

Comment:
`composeUp` does a `composeBuild` - see `buildBeforeUp` in https://github.com/avast/gradle-docker-compose-plugin